client/webserver/site/src/html/register.tmpl

```diff
@@ -71,6 +71,12 @@
           Enter your app password to confirm DEX registration.
           When you submit this form, <span id="feeDisplay"></span> DCR will be spent from your Decred wallet to pay registration fees.
         </div>
+        <div class="fs16 mt-4">
+          The DCR lot size for this DEX is <span id="dexDCRLotSize"></span> DCR.
+          All trades are in multiples of this lot size.
+          This is the minimum possible trade amount in DCR.
+          <!-- this will change when lot size is a market setting, not an asset setting -->
+        </div>
```

Comment:
This is bare minimum to resolve https://github.com/decred/dcrdex/issues/1041, but with the getdexinfo response, there can be a table of the markets and all the parameters, and other reg fee assets when we get there.